README.md

```diff
@@ -1,2 +1,149 @@
-# Workshop-Software-Architecture-Design
-A Founders &amp; Coders workshop, maintained by @lucyrose93
+# Introduction to Software Architecture Design
+
+### Learning Outcomes
+- To understand the general importance of software architecture design.
+- To start developing the critical ability to identify distinct elements of your app and their relations.
+- To apply this knowledge, and learn how you might begin architecting an app pre-build.
+
+### What Is Software Architecture Design?
+
+<iframe src="https://giphy.com/embed/85O5hFDD2xWeY" width="480" height="274" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/software-85O5hFDD2xWeY"></a></p>
+
+##### Architecture is the high-level / overall structure of a software system.
```

Comment:
Worth adding a link to a definition of 'high-level', or perhaps a brief explanation of 'abstraction'? Not all people start with a knowledge of it but the workshop revolves around so maybe worth making sure its clear to all.